File: sdkwork-backend-sdk-csharp/Models/AuthorInfo.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class AuthorInfo
    {
        public int? Id { get; set; }
        public ImageMediaResource? FaceImage { get; set; }
        public string? Name { get; set; }
        public string? Email { get; set; }
        public string? Bio { get; set; }
        public string? Website { get; set; }
    }
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/AuthorInfo.kt
package com.sdkwork.backend

data class AuthorInfo(
    val id: Int? = null,
    val faceImage: ImageMediaResource? = null,
    val name: String? = null,
    val email: String? = null,
    val bio: String? = null,
    val website: String? = null
)
